DEBIAN/control

```diff
@@ -0,0 +1,8 @@
+Package: tinygo
+Architecture: all
```

Comment:
Is this adjusted by `fpm`?
If not, I think `RELEASE_ARCH` from make should be used, similar to the version number.

Reply:
I assume we just ignore this for now.